Automattic/wp-calypso
# Onboarding Standalone Installation: Add new feature flag

#### Proposed Changes

* Add new `jetpack/standalone-plugin-onboarding-update-v1` feature flag for the new onboarding standalone installation.
* Add new copy to provide sample usage for the new feature flag.

#### Testing Instructions

1. * Run `git fetch && git checkout add/onboarding-standalone-installation-feature-flag`
    * Run `yarn start`
2. Go to http://calypso.localhost:3000/checkout/jetpack/thank-you/licensing-manual-activate-instructions/jetpack_backup_t1_yearly?flags=jetpack/standalone-plugin-onboarding-update-v1 or use the Calypso cloud live link and goto `/checkout/jetpack/thank-you/licensing-manual-activate-instructions/jetpack_backup_t1_yearly?flags=jetpack/standalone-plugin-onboarding-update-v1`.
3. Confirm the header displays **_Ok, let's install Jetpack Backup_.**
<img width="437" alt="Screen Shot 2022-10-14 at 5 02 36 PM" src="https://user-images.githubusercontent.com/56598660/195807736-81539b17-ad58-4084-a98b-d6d6a33f68e3.png">



#### Pre-merge Checklist






Related to 1203097885147382-as-1203159154260050

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203159154260048
  - https://app.asana.com/0/0/1203159154260050
